README.md

```diff
@@ -118,9 +112,11 @@ If the provider you're looking for hasn't been created yet, see the [develop a p
 Once you've added a provider as a dependency, it can be registered with OpenFeature like this:
 
 ```swift
-OpenFeatureAPI.shared.setEvaluationContext(evaluationContext: ctx)
+await OpenFeatureAPI.shared.setProviderAndWait(provider: MyProvider())
 ```
 
+> Asynchronous API that doesn't wait is also available
```

Comment:
I think "Asynchronous" is confusing here. The method isn't more or less synchronous than the normal setProvider? And also awaiting isn't blocking the thread either, so it's still async.

Reply:
`Asynchronous API that doesn't wait` -> actually refers to `setProvider()`. 
So yes, I'd say they are both "asynchronous" functions, but one allows to wait while the other is "fire-and-forget" (or "doesn't wait", as the docs say)